### DISCUSSION TITLE
Misskey mascot and visual identity

### DISCUSSION THREAD

**Discussion by gerardkieffer** [misskey-dev/misskey#9128] [Ideas]
I think that the "cat ears schoolgirl" mascot should be replaced in the misskey interface and on misskey-hub, by a more "serious" and grown up visual identity. It is an ambigous caracter which looks like a child and I feel like it might prevent the project from growing towords a broader audiance.
Just my personal oppinion, I'm eager to read yours :)

**Comment by ThatOneCalculator** [comment]
Ai-chan has been with Misskey since the beginning. Ai-chan supremacy 🫡

![image](https://user-images.githubusercontent.com/44733677/196248830-0afc25e9-c250-439b-8025-e201b0aedfe4.png)

**Comment by tassoman** [comment]
You have such a plenty of much moar mature softwearz to choose, that thing with ugly elephants, for example

**Comment by m33m33** [comment]
I explored this some time ago. Some ressources are hard coded, it's doable but won't last after an upgrade.
